.github/workflows/test.yml

```diff
@@ -21,13 +21,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go-version: [1.16.x, 1.17.x, 1.18.0-beta1]
+        go-version: [1.16.x, 1.17.x, 1.18.x]
```

Comment:
We can drop 1.16

Reply:
Good; modified accordingly. The change to drop 1.16 is somewhat big so implemented in a separate commit.